lib/index.ts

```diff
@@ -40,6 +41,17 @@ const DEFAULT_TTL = 600;
  */
 const ERROR_TTL = 1;
 
+/**
+ * If we find a value in Redis, how long should we recache it for in our RAM
+ * cache?
+ *
+ * Ideally we'd use the remaining TTL in Redis, and not this hard-coded value.
+ * But we have no easy way to get that. So we try to choose something shorter
+ * than most common TTLs, assuming that it's OK to periodically refetch this
+ * value from a Redis, which should have it with a longer TTL.
+ */
+const LOCAL_RECACHE_TTL = 15;
```

Comment:
why not use https://redis.io/commands/ttl?

Reply:
Oooh, that's a super good idea for the next version. :-)